### ISSUE TITLE
cp9 - WCT references should be CommunityToolkit.WinUI.UI.Controls...?

### ISSUE THREAD

**Issue by SolidRockProgrammer** [PacktPublishing/-Learn-WinUI-3.0#8]
Should the nugets be?
CommunityToolkit.WinUI.UI.Controls.DataGrid
and
CommunityToolkit.WinUI.UI.Controls

**Comment by SolidRockProgrammer** [comment]
ref
https://github.com/CommunityToolkit/WindowsCommunityToolkit/discussions/3912#discussioncomment-1210169
https://devblogs.microsoft.com/ifdef-windows/windows-community-toolkit-for-project-reunion-0-5/

**Comment by alvinashcraft** [comment]
Yes, it looks like some of the NuGet packages have changed since the book was published. I'm going to assign this to myself to look at when I create a branch for Windows App SDK v1.0

**Comment by AlexanderBlackman** [comment]
@alvinashcraft   Will you release an updated book for WinUI3 1.0? I'm holding off buying the March 2021 edition.

**Comment by alvinashcraft** [comment]
I would be interested in writing a second edition, updated to the final WinUI 3 version. I haven't been approached by Packt about it yet. You should let them know that you are interested.
